test/stdlib/TestCharacterSet.swift

```diff
@@ -192,6 +192,15 @@ class TestCharacterSet : TestCharacterSetSuper {
         expectEqual(expected, union)
     }
 
+    func test_subtractAndFormSymmetricDifference() {
```

Comment:
This is the 'happy path' test. What about subtracting an empty character set from a non-empty one and vice-versa?